src/timeline/Timeline.js

```diff
@@ -171,11 +171,14 @@ export default class Timeline extends React.PureComponent {
       } else {
         timeText = !format24h ? `${i - 12} PM` : `${i}:00`;
       }
+
+      const isTimeTextBeforeRestTime = this.props.restTime ? Number(moment(this.props.restTime.restEndTime).format("h")) >= Number(timeText.substring(0, 2)) : false;
```

Comment:
I think this should be resolved using offsets, not by parsing `timeText`. Time format may change and this might break

Reply:
Thanks! Fixed.